src/main/resources/java.gradle

```
@@ -10,6 +10,14 @@ tasks.withType(JavaCompile) {
 }
 
 repositories {
+    maven {
+        name = 'github'
+        url = uri('https://maven.pkg.github.com/clippings/service-core-starter')
```

Can we use base repo url in order to avoid changing this url or adding another repos if we need to resolve another artifacts in future (I am not very familiar with github packagies) 
```suggestion
        url = uri('https://maven.pkg.github.com/clippings/service-core-starter')
```

---

Unfortunately the answer is "No". The name of the repository is always `https://maven.pkg.github.com/OWNER/REPOSITORY`.

See
https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry
https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-gradle-registry